README.md

```diff
@@ -156,3 +143,22 @@ _You, too, can achieve perpetual fame and glory. Read some code, run some nodes,
 | @jrhea | `teku-attack-0`| [DoS Attack on Teku Stops Finality](https://github.com/ethereum/public-attacknets/issues/7) | $5k (USD) |
 | @jrhea | `lighthouse-attack-0`| [Network agent crashes lighthouse discovery](https://github.com/ethereum/public-attacknets/issues/4) | $1k (USD) |
 
+## Deprecated attacknets
+
+### Single-client `beta-0` attacknets
+
+_`beta-0` attacknets are deprecated. All nodes are have been disabled
+and attacks/rewards are no longer eligible._
+
+Single-client [beta-0 attacknets](./attacknets/beta-0) remain up, each with a continuous $5k bounty
+for novel exploits leading to sustained loss of finality.
+
+_Each network has a possible reward of $5k!_ [Read more](./attacknets/beta-0)
+for details about configuration, rules, and rewards.
+
+Deposits are not enabled in `beta-0` testnets so you'll have to try
+non-validator based attacks for this run.
+
+See README for each net for network configuration and rules/rewards.
+
```

Comment:
Can you make it more clear that this is the old text, and does not apply anymore, and the deprecation warning is what applies? Maybe quote it?

Reply:
will do